tests/jlm/hls/backend/rvsdg2rhls/DistributeConstantsTests.cpp

```diff
@@ -69,7 +68,6 @@ GammaSubregionUsage()
 
   // Assert
   assert(lambdaNode->subregion()->numNodes() == 2);
-  assert(constantNode.output(0)->IsDead());
```

Comment:
The old implementation also did a poor man's DNE. The new implementation does not try this, but rather relies on DNE to clean up afterwards.